bin/compile

```diff
@@ -30,7 +30,13 @@ curl --silent --retry 5 $BASE_URL/$SIGNATURE --output $SIGNATURE_PATH
 curl --silent --retry 5 $BASE_URL/$CHECKSUMS --output $CHECKSUMS_PATH
 
 echo "Verifying download" | indent
-gpg --quiet --keyserver pgp.mit.edu --recv 7BFB4EDA || gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+
+for i in 1 2 3 4; do
+  gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+  if [ $? = 0 ]; then
```

Comment:
The script is set to `set -e`, so doesn't look like the retry will ever happen.

Reply:
Yeah, this should explain why Travis failed twice in a row earlier today. Thanks for catching it and the fix in #11 :)